exec.go

```diff
@@ -210,5 +212,20 @@ func execute(params execParams) error {
 		return err
 	}
 
+	if !params.detach {
+		p, err := os.FindProcess(process.Pid)
+		if err != nil {
+			return err
+		}
+
+		ps, err := p.Wait()
```

Comment:
Right, currently all `exec`'s are "silently" detached but we never documented that (in the code or in an issue).